generate-component/index.js

```diff
@@ -0,0 +1,77 @@
+const fs = require("fs");
+const { component, story, test, barrel } = require("./component_templates.tsx");
+
+// grab component name from terminal argument
+const [name] = process.argv.slice(2);
+if (!name) throw new Error("You must include a component name.");
+
+const uppercasedName =
+  name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
```

Comment:
I was trying to make it easier for someone to be able to type a "lowercased" component name and have it be properly cased. But I guess if it's a Two worded component name (like ButtonGroup) it would break.

I'll just revert these changes and add a note telling people to just properly name the component. and use whatever is passed in